ts/client/storageEngine/transactions/constants.ts

```diff
@@ -0,0 +1,12 @@
+export enum Status {
+    // Transaction has been submitted to proposer
+    Pending = "pending",
+    // Transaction has been submitted as part of a batch
+    Submitted = "submitted",
+    // Transaction was finalized
+    Finalized = "finalized",
+    // Transaction was rolled back
+    RolledBack = "rolledback",
```

Comment:
Move to new issue covering transaction state after a rollback, remove from here.

Reply:
https://github.com/thehubbleproject/hubble-contracts/issues/585